.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+cache:
+  yarn: true
+# Send a message to MS Teams.
+notifications:
+  email: false
+  webhooks: https://outlook.office.com/webhook/03fa4a79-572f-4c68-b756-e4e851d0215a@9093f1a3-8771-4fb7-8596-d51eeef18cda/TravisCI/449087910d3647598cf3d7c6387fb8fc/286c079f-6085-4aa0-8f8d-e2a3e8d1f568
+node_js:
+  - '12' # mastarm 5 requires node.js 10 or greater
+# Copy example config for use in build.
+before_script:
+  - cp .env.tmp .env.build
```

Comment:
Does this accomplish anything in the now deployment?

Reply:
Yes, that file is read by the `next.config.js` script. It obviously doesn't have any variables for now, but we can address this by adding environment variables to Travis once we start running tests on Travis.

Reply:
Is that soon or should that be deferred?

Reply:
I say we remove that until we actually start using it.

Reply:
What I'm saying is that if that file does not exist, I believe the next.config.js will fail because it will try to read a non-existent file.